### PR TITLE
add support for new custom board - CircuitArt ESP32S3 zero

### PR DESCRIPTION
**Pull request by CircuitART**
This is a custom board that i developed it using the same footprint as pi zero.
the PID was obtained from Espressif https://github.com/espressif/usb-pids/pull/33
I have already build and tested circuitpython on it, and so far all is good

### CODE REVIEW

File: ports/espressif/boards/circuitart_zero_s3/mpconfigboard.mk [hunk 1]
```diff
@@ -0,0 +1,21 @@
+USB_VID = 0x303A
+USB_PID = 0x80DD
+USB_PRODUCT = "ZeroS3"
+USB_MANUFACTURER = "CircuiArt"
```
Comment (tannewt) [inline ports/espressif/boards/circuitart_zero_s3/mpconfigboard.mk:4]:
```suggestion
USB_MANUFACTURER = "CircuitArt"
```

**Review by tannewt — CHANGES_REQUESTED** [review]
What happened to the Feather S3 that you requested the PID for? It is weird you are reusing it.

**Comment by CircuitART** [comment]
my Feather S3 project was canceled and replaced with the zero version, I opened new pull request in espressif pid requesting changes on the pid name as well

**Review by tannewt — APPROVED** [review]
Ok, thanks!

**Comment by CircuitART** [comment]
@tannewt just thought to let u know know that Espressif-pid has been changed successfully
https://github.com/espressif/usb-pids/pull/179

**Comment by tannewt** [comment]
Thank you @CircuitART

**Pull request merged**